internal/sdkv2provider/resource_cloudflare_workers_kv.go

```diff
@@ -121,3 +122,7 @@ func parseId(id string) (string, string, error) {
 	}
 	return parts[0], parts[1], nil
 }
+
+func fmtId(namespaceID string, key string) string {
```

Comment:
this method is what [`parseId`](https://github.com/cloudflare/terraform-provider-cloudflare/blob/22331413ae48c47860d450f37f13cb931ab1182b/internal/sdkv2provider/resource_cloudflare_workers_kv.go#L117-L123) already does. the issue for the import is that it is expecting 3 but would only get two. i think we can revert everything outside of the `Import` changes and this will be fine.

Reply:
@jacobbednarz No problem. Done.